surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/fixture/HelperAssertions.java

```diff
@@ -159,4 +164,10 @@ public static IntegrationTestSuiteResults parseReportList( List<ReportTestSuite>
             throw new RuntimeException( "Couldn't parse XML reports", e );
         }
     }
+
+    public static void assumeJavaVersion(JavaVersion version)
+    {
+        assumeThat( "java.specification.version: ",
+                SystemUtils.JAVA_SPECIFICATION_VERSION, is( greaterThanOrEqualTo( version.toString() ) ) );
```

Comment:
I would not use JDK 9 for such simple comparison.
Back to `SystemUtils.JAVA_SPECIFICATION_VERSION, is( greaterThanOrEqualTo( version.toString() ) ) );`
It's matter of taste if you just compare strings or float numbers. Comparing float numbers would make me sleep better.